backend/src/auth_service/connection_package/persistence.rs

```diff
@@ -104,51 +127,97 @@ pub(crate) mod tests {
 
     use super::*;
 
-    async fn store_random_connection_packages_for_handle(
+    #[derive(Debug, Clone, Copy, PartialEq, Eq)]
+    pub(crate) enum ConnectionPackageType {
+        V1,
+        V2(bool), // is_last_resort
```

Comment:
Nit: Instead of a comment this could be just a named field. -- Feel free to ignore.